blog/2024-01-llms-graphs-part-2/index.md

```diff
@@ -117,8 +117,8 @@ SNW and HSNW instead are not exact indices. They are called approximate indices
 even approximate in the sense of having any approximation guarantees in their query results.
 They are heuristic-based indices that can index very high-dimensional vector and are fast 
 both in their construction and their query times. Further, their query results are shown to be quite accurate empirically.
-HNSW indices are nowadays used by vector database companies like [Pinecone](https://www.pinecone.io/learn/series/faiss/hnsw/)
-and [Weaviate](https://weaviate.io/developers/weaviate/concepts/vector-index) or enterprise search systems, such as [Lucene](https://lucene.apache.org/core/9_1_0/core/org/apache/lucene/util/hnsw/HnswGraph.html) in their vector indices.
+HNSW indices are nowadays used by vector databases like [Pinecone](https://www.pinecone.io/learn/series/faiss/hnsw/)
+and [Weaviate](https://weaviate.io/developers/weaviate/concepts/vector-index) or search engine libraries such as [Lucene](https://lucene.apache.org/core/9_1_0/core/org/apache/lucene/util/hnsw/HnswGraph.html) in their vector indices.
```

Comment:
Lucene is more a search engine library than a true search system, so a minor clarification in the terminology here.